crawler.py

```diff
@@ -12,8 +12,8 @@
 import time
 from shutil import copytree
 from urllib.request import urlopen
+from tranco import Tranco
 
-from PyFunceble import test as PyFunceble
```

Comment:
Should remove PyFunceble from requirements.txt and anywhere else it is referenced. Looks like there is a no-longer needed environment variable in the Dockerfile.